erdpy/tests/test_accounts_repository.py

```diff
@@ -1,12 +1,19 @@
-from erdpy.accounts import Address
+from typing import Protocol
+
+from erdpy.accounts import Address, AccountOnNetwork
 from erdpy.accounts_repository import AccountsRepository
-from erdpy.interfaces import IAddress, IElrondProxy
+from erdpy.interfaces import IAddress
 from erdpy.workstation import get_tools_folder
 
 TESTNET_USERS_FOLDER = get_tools_folder() / "testwallets" / "latest" / "users"
 DUMMY_NONCE = 42
 
 
+class INetworkProvider(Protocol):
+    def get_account_nonce(self, address: IAddress) -> int:
```

Comment:
This isn't implemented by erdpy-network-providers, if I remember well. Remove all calls to it (in erdpy)?

Reply:
And delete the `ElrondProxyStub` class as well?

Reply:
I think the stub has to stay, but to implement `get_account()`, instead?

```
repository.sync_nonces(proxy)
```

I think this function calls `get_account().nonce`, right? `get_account_nonce()` should normally not exist anymore in the code.

Reply:
Oh, I see you already did that.